HW_3/task_2.py

```diff
@@ -0,0 +1,29 @@
+"""
+Задайте список целых чисел. Верните список с произведениями его парных элементов.
+Парой считаются первый и последний элемент, второй и предпоследний и т.д.
+Если элементов нечетное количество – центральный элемент умножается сам на себя.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[2, 3, 4, 5, 6]
+[12, 15, 16]
+
+[2, 3, 5, 6]
+[12, 15]
+"""
+
+from random import randint
+
+number = int(input('Введите натуральное число N = '))
+my_list = [randint(0, 5) for _ in range(number)]
+print(my_list)
+
+my_list_produc = []
+for i in range(int((number) // 2) + 1):
```

Comment:
Во первых не нужна функция `int`, потому что целочисленное деление и уже целое число возвращает. Единицу нужно прибавлять перед делением.
```suggestion
for i in range((number + 1) // 2):
```